docs/index.md

```diff
@@ -2258,6 +2258,70 @@ Returns a job with the specified UUID.
       ]
     }
 
+## WaitJob (GET /jobs/:uuid/wait)
+
+Waits until the given job with the specified UUID is finished (i.e. when the
+job execution state is one of "succeeded", "failed" or "canceled").
```

Comment:
Does this mean that the request will not return a reply until the job is finished?

Reply:
Correct.